axiom/resource_notifier.go

```diff
@@ -81,11 +82,30 @@ type WebhookConfig struct {
 	URL types.String `tfsdk:"url"`
 }
 
+type CustomWebhookConfig struct {
+	URL     types.String                  `tfsdk:"url"`
+	Headers map[types.String]types.String `tfsdk:"headers"`
+	Body    types.String                  `tfsdk:"body"`
+}
+
 func (r *NotifierResource) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
 	resp.TypeName = req.ProviderTypeName + "_notifier"
 }
 
 func (r *NotifierResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	oneOfValidator := []validator.Object{
```

Comment:
Does this work?
I can't remember the exact reason as to why the validators could not contain the current attribute

Reply:
Have fixed!